src/util/gateway/index.ts

```diff
@@ -80,3 +85,17 @@ export interface DiscoveryQueueMatchPacket extends GatewayPacket {
 		channel: APIDMChannel;
 	};
 }
+
+export interface ClientTypingPacket extends GatewayPacket {
+	type: GatewayPacketType.ClientTyping;
+	data: {
+		channel: APIChannel;
```

Comment:
Can you make this channelID since the client shouldn't have to send the entire channel object.